<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="MainFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(242, 242, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="TopBlueCurve" native="true">
    <property name="geometry">
     <rect>
      <x>-100</x>
      <y>-230</y>
      <width>1400</width>
      <height>410</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.911, fx:0.495, fy:0.5, stop:0 rgba(124, 112, 255, 255), stop:0.522167 rgba(124, 112, 255, 255), stop:0.522685 rgba(255, 255, 255, 255), stop:0.527094 rgba(255, 255, 255, 0));
</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>650</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(222, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="fname_entry">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lname_entry">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>240</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="FName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>First Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>Profile Settings</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_entry">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>530</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>370</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(124, 112, 255);
color: rgb(255, 255, 255);
font: 87 14pt &quot;Segoe UI Black&quot;;
border-radius: 10px</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="FName_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>480</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>Old Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="FName_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>New Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mobile_entry">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>320</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Images/arroe_left.png</normaloff>../Images/arroe_left.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="FName_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>600</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>Confirm Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>Change Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="FName_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_password">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>640</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(124, 112, 255);
color: rgb(255, 255, 255);
font: 87 14pt &quot;Segoe UI Black&quot;;
border-radius: 10px</string>
    </property>
    <property name="text">
     <string>Change Password</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>430</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_entry">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="SName_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>330</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>Mobile Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="oldpassword_entry">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>470</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="SName">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>Second Name: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="repassword_entry">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>590</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>200</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>360</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Images/Logo.png</normaloff>../Images/Logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>350</width>
      <height>350</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
